src/main/scala/io/iohk/ethereum/cli/CliCommands.scala

```diff
@@ -0,0 +1,74 @@
+package io.iohk.ethereum.cli
+
+import cats.implicits._
+import cats.data.NonEmptyList
+import com.monovore.decline.{Command, Opts}
+import io.iohk.ethereum.crypto
+import io.iohk.ethereum.crypto._
+import io.iohk.ethereum.utils.ByteStringUtils
+import java.security.SecureRandom
+import org.bouncycastle.util.encoders.Hex
+
+object CliCommands {
+
+  val generatePrivateKeyCommand = "generate-private-key"
+  val deriveAddressCommand = "derive-address"
+  val generateAllocsCommand = "generate-allocs"
+  val useAddressesFlag = "useAddresses"
```

Comment:
why not replace it with either `--address=<address>` or `--key=<key` (or in plural)? In that way it would be a bit easier and always clear what is passed without reaching for bool flags